java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2758,14 +2759,63 @@ public T zrangeWithScores(@NonNull String key, @NonNull ScoredRangeQuery rangeQu
         return zrangeWithScores(key, rangeQuery, false);
     }
 
-    // TODO add @link to ZMPOP when implemented
+    /**
+     * Pops a member-score pair from the first non-empty sorted set, with the given <code>keys</code>
+     * being checked in the order they are provided.
+     *
+     * @since Redis 7.0 and above.
+     * @see <a href="https://redis.io/commands/zmpop/">redis.io</a> for more details.
+     * @param keys The keys of the sorted sets.
+     * @param modifier The element pop criteria - either {@link ScoreFilter#MIN} or {@link
+     *     ScoreFilter#MAX} to pop members with the lowest/highest scores accordingly.
+     * @return Command Response - A two-element <code>array</code> containing the key name of the set
+     *     from which an element was popped, and a member-score <code>Map</code> of the popped
+     *     elements.<br>
```

Comment:
Same nit suggestion as in SortedSetBaseCommands
```suggestion
     *     ScoreFilter#MAX} to pop the member with the lowest/highest score accordingly.
     * @return Command Response - A two-element <code>array</code> containing the key name of the set
     *     from which the element was popped, and a member-score <code>Map</code> of the popped
     *     element.<br>
```